### PR TITLE
BAU - Switch to jakarta for rest-utils and stop supporting java8

### PR DESCRIPTION
**Pull request by JHjava**
- Upgrade gradle to the latest version 
- Stop supporting java8 as we have no applications in Verify that still use java8. 
- Stop Travis from compiling against java8
- Update rest-utils to use the jakarta dependency as the the javax package namespace is no longer supported and all releases will be done under the jakarta namespace.
- Bring in glassfish jersey-server to rest-utils so we can use the latest rather than the version that the dropwizard dependency brings in.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ env:
 
 jdk:
   - openjdk11
```
Comment (chrisclayson) [inline .travis.yml:12]: Is it worth adding Java 15?

Comment (chrisclayson) [reply .travis.yml:12]: Although, I get build failures with JDK 15!

**Review by chrisclayson — APPROVED** [review]
Looks OK

File: .travis.yml (hunk 1 above)
Comment (JHjava) [reply .travis.yml:12]: Yep this repo is a bit neglected. If I get time I'm planning on making it better

**Review by dbes-gds — APPROVED** [review]

**Pull request merged**